packages/devtools_app/lib/src/shared/console/widgets/description.dart

```diff
@@ -553,13 +563,49 @@ class DiagnosticsNodeDescription extends StatelessWidget {
   }
 }
 
+/// Label for a property with the default value.
+class DefaultValueLabel extends StatelessWidget {
+  const DefaultValueLabel({
+    super.key,
+    required this.colorScheme,
+    required this.textStyle,
+  });
+
+  final ColorScheme colorScheme;
+  final TextStyle textStyle;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.symmetric(
+        horizontal: denseSpacing,
+      ),
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(7.5),
```

Comment:
could we use `borderRadius: defaultBorderRadius` here instead?

Reply:
Yes! Switched to that